src/QsCompiler/Tests.LanguageServer/ProjectLoaderTests.cs

```diff
@@ -129,10 +129,10 @@ public void LoadNonQsharpProjects()
         public void LoadOutdatedQsharpProject()
         {
             var (projectFile, context) = this.Context("test9");
-            var projDir = Path.GetDirectoryName(projectFile);
+            var projDir = Path.GetDirectoryName(projectFile) ?? "";
```

Comment:
```suggestion
           var projDir = Path.GetDirectoryName(projectFile) ?? string.Empty;
```

Reply:
I think using string.Empty is slightly better since it expresses intent and the person that is looking at the code can be sure it was not a typo or something similar.

Reply:
I thought it was a just these two lines but it seems like there are more like this one. Please let me know if you agree with this change before going ahead and making a bunch of suggestions that might end up not being integrated :P

Reply:
I prefer `""`, since it's shorter and consistent with non-empty strings.

Reply:
Ok, I don't have a string preference for string.Empty so I'll close the suggestions regarding `""` vs `string.Empty`.